com.avaloq.tools.ddk.xtext.ide/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,13 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: com.avaloq.tools.ddk.xtext.ide
+Bundle-SymbolicName: com.avaloq.tools.ddk.xtext.ide;singleton:=true
+Bundle-Version: 8.3.0.qualifier
+Bundle-Vendor: Avaloq Evolution AG
+Require-Bundle: org.eclipse.xtext.ui,
+ org.eclipse.lsp4j,
+ com.avaloq.tools.ddk.xtext
```

Comment:
Why is  com.avaloq.tools.ddk.xtext needed? I do not see any dependency in the code for this one.